tests/suites/cli/local_charms.sh

```diff
@@ -1,26 +1,30 @@
+# Copyright 2024 Canonical Ltd.
+# Licensed under the AGPLv3, see LICENCE file for details.
+
 # Checks whether the cwd is used for the juju local deploy.
 run_deploy_local_charm_revision() {
 	echo
 
 	file="${TEST_DIR}/local-charm-deploy-git.log"
 	ensure "local-charm-deploy" "${file}"
 
+	# Get a basic charm
 	TMP=$(mktemp -d -t ci-XXXXXXXXXX)
+	cp -r "$CURRENT_DIR/../testcharms/charms/ubuntu-plus" "${TMP}"
```

Comment:
You technically don't need to use CURRENT_DIR here, elsewhere we just navigate from cwd

e.g.
https://github.com/juju/juju/blob/3.6/tests/suites/hooks/dispatch.sh#L13

Reply:
The problem is these tests cd around, I was just doing that before and it kept failing on subsequent tests

Reply:
It's probably better to do it this way, explicit is better.